topology/pre-processor.c

```diff
@@ -162,6 +164,10 @@ static int pre_process_config(struct tplg_pre_processor *tplg_pp, snd_config_t *
 		}
 	}
 
+	/* create nhlt blob and add it to manifest */
+	if (nhlt_process)
```

Comment:
I'm not sure it is a good idea to have the flag set during compilation. I think this should come from the conf file itself. For example, when processing the DMIC object, we should somehow be able to say add NHLT blob for this DMIC

Reply:
hmm ok... yes could be done. let me try it out